src/components/page/body.vue

```diff
@@ -10,10 +10,8 @@ including this file, may be copied, modified, propagated, or distributed
 except according to the terms contained in the LICENSE file.
 -->
 <template>
-  <div id="page-body" class="row">
```

Comment:
We need to add margins to the element, but `.row` already has margins. And actually, I don't think this element even needs to use `.row`: I don't think `.row` + `.col-xs-12` has much of an effect.